reactor-test/src/main/java/reactor/test/DefaultStepVerifierBuilder.java

```diff
@@ -2525,7 +2525,7 @@ static String formatFusionMode(int m) {
 				Scannable lowest = Scannable.from(s);
 
 				//attempt to go back to the leftmost parent to check the Context from its perspective
-				Context c = Flux.<Scannable>
+				CoreSubscriber<?> subscriber = Flux.<Scannable>
```

Comment:
wait, do we still need that? now the `subscriber` could be `null` and none of the `Consumer<CoreSubscriber>` have been amended to fix that :(

Reply:
We do. Also, there is a check that verifies that the context is present before running any of the context assertions

Reply:
yeah just re-checked the code and tests, this should be equivalent, as previously this resulting in a `Context` variable that was also `null`. My bad.